mixes/containers/mix.js

```diff
@@ -7,25 +7,32 @@ const { loadMix, saveMix } = require('../actions')
 class MixContainer extends React.Component {
   render () {
     const { mix, isLoading, isSaving, error, saveMix } = this.props
+    if (!mix) { return null }
+
+    console.log('mix', mix)
 
     return <div>
       <header>
-        <div>mix is {isLoading ? 'loading' : 'here'}</div>
+        <div>'{mix.meta.title}' is {isLoading ? 'loading' : 'here'}</div>
         <div>{error ? error : 'no errors'}</div>
         <button disabled={isLoading || isSaving} onClick={() => saveMix(mix)}>Save Mix</button>
       </header>
-      <section>
-        {mix.id}
+      <section key={mix.id}>
+        {mix.primaryTracks.map(track => {
+          return <div key={track.id} >{track.meta.title}</div>
```

Comment:
when to use section vs div? i discern that `key` is used by react to track which DOM nodes are changing. is it possible to render two different elements with the same `key` from two different components? eg if this `mix` container were rendered twice on the same page